### PR TITLE
Fixes part of #2215: Changes in Fragments under Options Menu [A11y]

### PR DESCRIPTION
**Pull request by Mehul-Bisht**
## Explanation

Fixes part of #2215: Changes in Fragments under Options Menu [A11y]
Fixes the accessibility issues with Reading Text Size Fragment

## Reading Text Size Fragment
| Before | After |
:-------------------------:|:-------------------------:
before |  after

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/reading_text_size_fragment.xml [hunk 1]
```diff
@@ -67,8 +68,8 @@
 
           <TextView
             android:id="@+id/reading_text_size_small_textview"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="10dp"
+            android:layout_height="17dp"
```
Comment (Arjupta) [inline app/src/main/res/layout/reading_text_size_fragment.xml]: @Mehul-Bisht can you explain the need for this change

Comment (Mehul-Bisht) [reply app/src/main/res/layout/reading_text_size_fragment.xml]: @ARJUPTA seems that I was checking something and forgot to revert this change, I'll do that now.

**Comment by Mehul-Bisht** [comment]
@ARJUPTA kindly review again

**Comment by Arjupta** [comment]
@rt4914 PTAL.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -416,6 +416,6 @@
     description="The default hint text for the ratio interaction if the placeholder text is not available.">
     Enter a ratio in the form x:y.
   </string>
-
-
+  <string name="smallest_textsize">Smallest textSize</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Suggested change:
`<string name="smallest_text_size_content_description">Smallest text size</string>`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -416,6 +416,6 @@
...
+  <string name="largest_textsize">Largest textSize</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Suggested change:
`<string name="largest_text_size_content_description">Largest text size</string>`

**Review by rt4914 — CHANGES_REQUESTED** [review]
The solution does work but whatever stings we use for `content-description` should be verified by @mschanteltc 

@mschanteltc In this [image](https://user-images.githubusercontent.com/64064110/101000846-69af8d80-3584-11eb-89be-0a227ec4bca6.jpg) the Accessibility Scanner is failing because both `A` (the smallest and the largest) have same content description i.e. `A`. So this PR solves that by changing the content description to following lines:
1. For smallest `A`: `Smallest text size`
2. For larges `A`: `Largest text size`.

What are you suggestions and what should be the correct content description?

**Comment by rt4914** [comment]
cc @ARJUPTA https://github.com/oppia/oppia-android/pull/2594#pullrequestreview-582038019

**Comment by Mehul-Bisht** [comment]
@rt4914 @mschanteltc once finalised, kindly let me know about your decision.

**Comment by mschanteltc** [comment]
> The solution does work but whatever stings we use for `content-description` should be verified by @mschanteltc
> 
> @mschanteltc In this [image](https://user-images.githubusercontent.com/64064110/101000846-69af8d80-3584-11eb-89be-0a227ec4bca6.jpg) the Accessibility Scanner is failing because both `A` (the smallest and the largest) have same content description i.e. `A`. So this PR solves that by changing the content description to following lines:
> 
> 1. For smallest `A`: `Smallest text size`
> 2. For larges `A`: `Largest text size`.
> 
> What are you suggestions and what should be the correct content description?

The content description you suggested should work. Are we able to provide the font sizes of these A's, or would it conflict with the device's text size settings?

**Comment by rt4914** [comment]
> > The solution does work but whatever stings we use for `content-description` should be verified by @mschanteltc
> > @mschanteltc In this [image](https://user-images.githubusercontent.com/64064110/101000846-69af8d80-3584-11eb-89be-0a227ec4bca6.jpg) the Accessibility Scanner is failing because both `A` (the smallest and the largest) have same content description i.e. `A`. So this PR solves that by changing the content description to following lines:
> > 
> > 1. For smallest `A`: `Smallest text size`
> > 2. For larges `A`: `Largest text size`.
> > 
> > What are you suggestions and what should be the correct content description?
> 
> The content description you suggested should work. Are we able to provide the font sizes of these A's, or would it conflict with the device's text size settings?

@mschanteltc No we cannot provide font size because it will create conflict with the device font size too. For example: small A on one device can be 14 units and on another it can be 20 units.

**Comment by mschanteltc** [comment]
For the A's let's use the description that you suggested which is "Smallest text size" and "Largest text size." Are content descriptions needed for each interval when the cursor moves?

**Comment by rt4914** [comment]
> For the A's let's use the description that you suggested which is "Smallest text size" and "Largest text size." Are content descriptions needed for each interval when the cursor moves?

@mschanteltc Not sure but that would be part of a different PR because that's something very different and does not fall under **Accessibility Scanner**. It will come under **Talkback**. Thanks.

**Comment by rt4914** [comment]
@Mehul-Bisht This is confirmed:

```
For smallest A: `Smallest text size`
For largest A: `Largest text size`
```

You can update the overall PR. Thanks.

**Comment by Mehul-Bisht** [comment]
@rt4914 Updated as per the changes suggested, kindly merge and close this PR now.

**Comment by Mehul-Bisht** [comment]
@rt4914 any updates on this PR?

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/values/strings.xml]: @Mehul-Bisht You have missed this suggestion

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (rt4914) [reply app/src/main/res/values/strings.xml]: @Mehul-Bisht You have missed this suggestion

**Comment by rt4914** [comment]
> @rt4914 any updates on this PR?

@Mehul-Bisht Two of my comments have not been solved. PTAL and update the PR.

**Comment by Mehul-Bisht** [comment]
@rt4914 I updated now, please check

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**